sklearn/mixture/gaussian_mixture.py

```diff
@@ -648,3 +672,22 @@ def aic(self, X):
             The greater the better.
         """
         return -2 * self.score(X) * X.shape[0] + 2 * self._n_parameters()
+
+    @property
+    def precisions_(self):
+        self._check_is_fitted()
+        if self.precision_type is 'full':
+            return np.array([np.dot(prec, prec.T)
+                            for prec in self._precisions])
+        elif self.precision_type is 'tied':
+            return np.dot(self._precisions, self._precisions.T)
+        else:
+            return self._precisions
+
+    @property
+    def precisions_cholesky_(self):
+        self._check_is_fitted()
+        if self.precision_type in ['full', 'tied']:
+            return self._precisions
+        else:
+            return np.sqrt(self._precisions)
```

Comment:
> to me we should not change the public API just store as attributes what is
necessary for fast prediction (for cov models)

Note that the new GaussianMixture class has never been released yet so we don't have backward compat issue (yet) on this class.

> If we do that we will apply sqrt + square at each step. I know that it's not really time consuming but I don't like apply useless function just to remove a private var.

I think this is worth it: it makes the code simpler to read.